playground/frontend/lib/constants/api.dart

```diff
@@ -16,4 +16,5 @@
  * limitations under the License.
  */
 
-const String kApiClientURL = 'https://datatokenization.uc.r.appspot.com';
+const String kApiClientURL =
+    'https://backend-dot-datatokenization.uc.r.appspot.com';
```

Comment:
May we consider not hardcoding the URL in the codebase?  Apologies that I missed it in an earlier review.  One possible way to solve it is using a `config.json` file like [this](https://github.com/damondouglas/fs-auth-example/tree/main/frontend/assets#configjson) and [this](https://github.com/damondouglas/fs-auth-example/blob/main/frontend/lib/config.dart).  The friction, however, is that it requires creating and maintaining this file.  Perhaps we can add a gradle task that creates this config, acquiring the endpoint URL from a gradle.properties file in `playground/frontend/gradle.properties`.

Reply:
@damondouglas, thank you for the suggestion. I pushed a task to generate `config.g.dart` file, could you please check it?
I chose to generate a dart config file, because it's easier to use in code.